rudder-agent/SOURCES/check-rudder-agent

```diff
@@ -60,6 +60,25 @@ clean_cf_lock_files() {
   rm -f ${CFE_DIR}/state/cf_lock.${CFENGINE_DB_EXT}.lock
 }
 
+fix_perm() {
+  perm="$1"
+  file="$2"
+  if [ `stat -c %a "${file}"` != "${perm}" ]; then
```

Comment:
I think it's probably not worth it, because I'm pretty sure the chmod command itself already does a similar test to avoid unnecessary writes.